core/src/main/java/org/apache/gravitino/storage/relational/JDBCBackend.java

```diff
@@ -360,6 +363,39 @@ public List<TagEntity> associateTagsWithMetadataObject(
         .associateTagsWithMetadataObject(objectIdent, objectType, tagsToAdd, tagsToRemove);
   }
 
+  @Override
+  public <E extends Entity & HasIdentifier> List<E> listEntitiesByRelation(
+      SupportsRelationOperations.Type relType,
+      NameIdentifier nameIdentifier,
+      Entity.EntityType identType) {
+    if (relType == SupportsRelationOperations.Type.OWNER_REL) {
+      List<E> list = Lists.newArrayList();
+      OwnerMetaService.getInstance()
+          .getOwner(nameIdentifier, identType)
+          .ifPresent(e -> list.add((E) e));
+      return list;
+    } else {
+      throw new IllegalArgumentException(
+          String.format("Doesn't support the relation type %s", relType));
+    }
+  }
```

Comment:
Using switch case, not if else here and below.

Reply:
OK, IDEA suggested I use `if else`. It's ok for me to use switch.

Reply:
Changed.